recipe/meta.yaml

```
@@ -1,75 +1,61 @@
 {% set name = "python-ldap" %}
-{% set version = "3.4.0" %}
+{% set version = "3.4.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 60464c8fc25e71e0fd40449a24eae482dcd0fb7fcf823e7de627a6525b3e0d12
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: b2f6ef1c37fe2c6a5a85212efe71311ee21847766a7d45fcb711f3b270a5f79a
 
 build:
-  number: 1
+  number: 0
   skip: true  # [win]
```

```suggestion
  skip: true  # [win or py<36]
```

https://github.com/python-ldap/python-ldap/blob/python-ldap-3.4.5/pyproject.toml#L17

---

And btw - why it's skipped on Windows anyway?

https://github.com/python-ldap/python-ldap/blob/python-ldap-3.4.5/pyproject.toml#L25

---

> And btw - why it's skipped on Windows anyway?

Inherited from the previous release. They did not change it for the last 3.5 years (git-blame). We can definitely update it.

---

Looks like `openldap` is not available on Windows :) I added a comment on that.